automatic/azcopy10/azcopy10.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>azcopy10</id>
+    <version>10.21.1</version>
+    <title>azcopy10</title>
+    <authors>Microsoft</authors>
+    <owners>jbpaux, pauby</owners>
+    <projectUrl>https://github.com/Azure/azure-storage-azcopy</projectUrl>
+    <packageSourceUrl>https://github.com/jbpaux/chocolatey-packages/tree/master/azcopy10</packageSourceUrl>
+    <iconUrl>https://rawcdn.githack.com/jbpaux/chocolatey-packages/264e0172dcd810bc7ea08a2225876cff9710aa6f/azcopy10/azcopy-logo.png</iconUrl>
```

Comment:
This URL too

Reply:
I've updated this, but I'm not 100% sure on how it'll work. I've updated it to a commit that's part of this PR, but if the commits change at all before merge then I don't think it'll be valid.

Reply:
`iconUrl` has been updated to `https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@de115ff0e16d124c2099b8f95bd08bb92b6e4918/icons/azcopy10.png`. I'm going to resolve this conversation.